tests/python/test_genotype_extractor_worker.py

```diff
@@ -0,0 +1,144 @@
+        with tables(db.engine, 'vcfs', 'genotypes') as (con, vcfs, genotypes):
+            vcf = vcfs.select(vcfs.c.id == self.run['id']).execute().fetchone()
+            variants = genotypes.select(
+                genotypes.c.vcf_id == self.run['id']).execute().fetchall()
+            variants = [dict(v) for v in variants]
+        expected_extant_columns = set(["sample:GT", "sample:RD", "sample:FREQ",
+                                       "info:SSC", "sample:DP", "sample:DP4",
+                                       "info:DP", "info:SS", "info:GPV", "sample:AD",
+                                       "info:SPV"])
+        expected_vcf_header = """\
+##fileformat=VCFv4.1
+##source=VarScan2
+##INFO=<ID=DP,Number=1,Type=Integer,Description="Total depth of quality bases">
+##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description="Indicates if record is a somatic mutation">
+##INFO=<ID=SS,Number=1,Type=String,Description="Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)">
+##INFO=<ID=SSC,Number=1,Type=String,Description="Somatic score in Phred scale (0-255) derived from somatic p-value">
+##INFO=<ID=GPV,Number=1,Type=Float,Description="Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls">
+##INFO=<ID=SPV,Number=1,Type=Float,Description="Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls">
+##FILTER=<ID=str10,Description="Less than 10% or more than 90% of variant supporting reads on one strand">
+##FILTER=<ID=indelError,Description="Likely artifact due to indel reads at this position">
+##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
+##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="Genotype Quality">
+##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read Depth">
+##FORMAT=<ID=RD,Number=1,Type=Integer,Description="Depth of reference-supporting bases (reads1)">
+##FORMAT=<ID=AD,Number=1,Type=Integer,Description="Depth of variant-supporting bases (reads2)">
+##FORMAT=<ID=FREQ,Number=1,Type=String,Description="Variant allele frequency">
+##FORMAT=<ID=DP4,Number=4,Type=Integer,Description="Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev">
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NORMAL	TUMOR"""
+        extant_columns = set(ast.literal_eval(vcf['extant_columns']))
+        asserts.eq_(extant_columns, expected_extant_columns)
+        # 10 variants x 2 for the two samples in this file, NORMAL & TUMOR
+        asserts.eq_(vcf['genotype_count'], 20)
+        asserts.eq_(vcf['genotype_count'], len(variants))
+        asserts.eq_(vcf['vcf_header'], expected_vcf_header)
+
+        vcfreader = pyvcf.Reader(open('tests/data/snv.vcf'))
+        expected_variants = [r for r in vcfreader]
+
+        # Make sure we have the same (chromosome, position, ref, alt) variants
+        assert_keys_in_variants(expected_variants, variants)
+
+        # Make sure we've got the INFO fields we expect
+        expected_extractors = [lambda v: v.INFO['DP'],
```

Comment:
I got confused and thought you were literally trying to compare extractors (the "expected" vs. "actual" extractor). Maybe call this `pyvcf_extractors` or something?